web/modules/custom/dbcdk_community/src/Plugin/Block/ProfileQuarantinesBlock.php

```diff
@@ -96,23 +85,14 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function build() {
-    $context_username = $this->getContextValue('username');
-    $quarantines = [];
     try {
-      // Fetch a Community Profile based on the username so we can the
-      // corresponding ID to fetch quarantines.
-      $profile_filter = [
-        'where' => [
-          'username' => $context_username,
-        ],
-      ];
-      $profile = $this->profileApi->profileFindOne(json_encode($profile_filter));
-
-      $quarantines = (array) $this->profileApi->profilePrototypeGetQuarantines($profile->getId(), json_encode(['order' => 'end DESC']));
+      $profile = $this->profileRepository->getProfileByUsername($this->getContextValue('username'));
     }
     catch (ApiException $e) {
       \Drupal::logger('DBCDK Community Service')->error($e);
-      $profile = NULL;
+      $profile = new Profile();
+      // The generated Profile model does not have a public quarantines
+      // attribute but the repository will add it when retrieving the object.
```

Comment:
Is this a misplaced comment or leftover from earlier code that did not make the cut at the end?